hack/update-proto.sh

```diff
@@ -18,6 +18,9 @@ HACK_DIR=$(dirname "${BASH_SOURCE}")
 
 echo "Updating plugin proto"
 
+protoc pkg/plugin/proto/backupitemaction/v1/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ -I pkg/plugin/proto/
 protoc pkg/plugin/proto/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ -I pkg/plugin/proto/
```

Comment:
For the above BIA code to generate properly locally, I had to add go_package to Shared.proto. For this protoc command to work on the other top level unversioned files, I needed that as well. Once added, the this protoc line needs the same `--go_opt=module=github.com/vmware-tanzu/velero/pkg/plugin/generated` as the first one. The required go_package line to all files in `pkg/plugin/generated` is this:
```
option go_package = "github.com/vmware-tanzu/velero/pkg/plugin/generated";
```